data/json/effects.json

```diff
@@ -1011,7 +1011,7 @@
         "type": "effect_type",
         "id": "common_cold",
         "name": ["Common Cold"],
-        "desc": ["Symptoms alleviated by medication (cough syrup)."],
+        "desc": ["You have a common cold.  Symptoms can be alleviated by medication (cough syrup)."],
```

Comment:
I'd recommend `"You have contracted the 'Common Cold'.` or you could even be more technical as, `"You have contracted rhinitis.`